src/main/java/br/com/excaladashboard/models/Cliente.java

```diff
@@ -16,20 +14,76 @@ public class Cliente {
     @Column(name = "id")
     private Long id;
 
+    @Column(name = "ativo", length = 256)
+    private String ativo;
+
     @Column(name = "empresa", length = 256)
     private String empresa;
 
-    @Column(name = "responsavel", length = 256)
-    private String responsavel;
+    @Column(name = "telefone", length = 256)
+    private String telefone;
+
+    @Column(name = "email", length = 256)
+    private String email;
+
+    @Column(name = "cnpj", length = 256)
+    private String cnpj;
+
+    @Column(name = "cpf", length = 256)
+    private String cpf;
+
+    @Column(name = "pagamento_em_dia", length = 256)
+    private String pagamentoDia;
+
+    @Column(name = "contrato_assinado", length = 256)
+    private String contratoAssinado;
```

Comment:
aqui acho melhor colocar como valor boolean também

```suggestion
    @Column(name = "contrato_assinado")
    private boolean contratoAssinado;
```